CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 See below for Changelog examples.
 
+## 0.7.0
```

Comment:
The release number shouldn't be added to the changelog except when doing a release